### ISSUE TITLE
[1.18.2]-5.2.2 Charging station only work when plane is moving

### ISSUE THREAD

**Issue by TheTrueSCP** [przemykomo/simple-planes#176]
The charging station only works when the plane is moving. I think, regarding to the source-code, the plane only have the energy capabillity when it drains the energy tank. Thats why the charging station tile entity only find the plane when its moving, at least i think so...

Forge Version: 20.2.10

**Comment by OutcastZeroOne** [comment]
no, it charges but it wont display on the plane till you try to start it. its a glitch but it is charging.
